example/workspace/Coffe.ecore

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where do we use this file?

Reply:
I simply copied the test workspace from the modelserver repo.
I don't think it is used (yet)

Reply:
Ok, I just ask because it seems as if the file is outdated (should be Coffee.ecore).